Menu/Provider/GroupMenuProvider.php

```diff
@@ -147,9 +147,9 @@ public function get($name, array $options = array())
                 if (isset($item['admin']) && !empty($item['admin'])) {
                     $admin = $this->pool->getInstance($item['admin']);
 
-                    // skip menu item if no `list` url is available or user doesn't have the LIST access rights
+                    // Return null if no `list` url is available or user doesn't have the LIST access rights
                     if (!$admin->hasRoute('list') || !$admin->hasAccess('list')) {
-                        continue;
+                        return null;
```

Comment:
What if there is another item in the group, for which you have rights, but it's in second position? You will return `null` in that case, right? This looks wrong…